### ISSUE TITLE
creating SNANA simlib from a light curve file

### ISSUE THREAD

**Issue by djones1040** [RickKessler/SNANA#1349]
Hi Rick!  @jpierel14 and I are trying to make a simple simulation of some JWST observations.  I vaguely remembered that we can construct a basic SIMLIB from lightcurve data and had an example how this is supposed to work from old SALT3 stuff.  But I'm now getting an error that I don't totally understand.  The files live in `$SNANA_DEBUG/djones/JWST` and you can reproduce with:

```
snana.exe snfit_JWST.nml HEADER_OVERRIDE_FILE jwst.FITRES.TEXT SIMLIB_OUT jwst.simlib
```
and then I get the error:
```
 init_obs_atFLUXMAX:
       SET_PEAKMJD(tr27) --> 60231.164   (FluxMax+BazinFit)
       Analyze ISN_PROC=       1  for tr27            NMJD(stored,cuts)=  15->  15
       SNLCPLOT: pack CID=tr27 for plotting.

   `|```````|`
   <| o\ /o |>
    | ' ; ' |
    |  ___  |     ABORT program on Fatal Error.
    | |' '| |
    | `---' |
    \_______/

 FATAL ERROR ABORT called by quickBinSearch
   VAL = 3.559325e+04 outside 'COMPUTE_SIMLIB_INFO ' range
   defined for 1.000000e+03 < VAL < 2.000000e+04

```
Any insight into where this is coming from and how we might fix it?  Thanks!

**Comment by RickKessler** [comment]
Indeed that abort message is rather cryptic, so here is a more clear error message:

 FATAL ERROR ABORT called by COMPUTE_SIMLIB_INFO
   LAMOBS=  35593. exceeds max LAM=  20000.
   Need to extend SKYLAM and SKYMAG arrays in SIMLIBCOM

In snana.car, will need to extend hard-wired
SKYLAM_SPACE_LIST
SKYMAG_SPACE_LIST    # guess at sky birghtness (mag/arcsec^2)
to go past L & M bands.

**Comment by RickKessler** [comment]
Just merged PR with extension for SKYLAM_SPACE_LIST.

**Comment by RickKessler** [comment]
yes, go ahead and extend ground based upper limit to 30,000; then send a PR.

**Comment by RickKessler** [comment]
Done.
